thirdparty/google-styleguide/run_linter.py

```diff
@@ -16,7 +14,8 @@
       if (".h" in file_name) or (".cpp" in file_name):
         list_of_files += [os.path.join(dir_path, file_name)]
 
-repository_path = script_dir + "../../"
+script_dir = os.path.dirname(os.path.realpath(__file__))
+repository_path = os.path.abspath(os.path.join(script_dir, '..', '..'))
```

Comment:
Nitpicking: "os.pardir" is a platform-independent solution, so I would suggest to use that instead of ".."

Reply:
Changed it :)